settings.gradle

```diff
@@ -25,6 +25,7 @@ pluginManagement {
 rootProject.name = 'openhouse'
 
 include ':apps:spark'
+include ':apps:spark-3.5'
```

Comment:
Can we also rename this module?

Reply:
when i attempt to rename `:apps:spark-3.5`, to `:apps:openhouse-spark-apps-1.5_2.12`

```bash
A problem occurred evaluating project ':apps:openhouse-spark-apps-1.5_2.12'.
> Could not get unknown property 'sourceSets' for project ':apps:openhouse-spark-apps_2.12' of type org.gradle.api.Project.
```

which is... unexpected. i will leave the module the same name then because I attempted a few different alternatives without success

Reply:
found the issue... gradle was unable to automatically make the two projects depend on each other. fixed